viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -736,6 +736,13 @@ export class Cognite3DViewer {
       preMultipliedRotation
     );
 
+    const numberOf360Images = image360Collection.image360Entities.length;
+
+    MetricsLogger.trackEvent('360ImageCollectionAdded', {
```

Comment:
If `numberOf360Images` is `0`, should we track the event as `360ImageCollectionAdded`